tests/PKSim.Tests/CLI/ExportSimulationRunnerSpecs.cs

```diff
@@ -182,13 +182,13 @@ protected override Task Because()
       [Observation]
       public void should_run_the_export_for_the_selected_simulations_defined_in_the_project()
       {
-         A.CallTo(() => _simulationExporter.Export(_simulation1, _s1OutputFolder,  _exportRunOptions.ExportMode)).MustHaveHappened();
+         A.CallTo(() => _simulationExporter.Export(_simulation1, _s1OutputFolder,  _exportRunOptions.ExportMode, null)).MustHaveHappened();
       }
 
       [Observation]
       public void should_not_run_the_export_for_the_simulation_excluded_from_the_run()
       {
-         A.CallTo(() => _simulationExporter.Export(_simulation2, _s2OutputFolder,  _exportRunOptions.ExportMode)).MustNotHaveHappened();
+         A.CallTo(() => _simulationExporter.Export(_simulation2, _s2OutputFolder,  _exportRunOptions.ExportMode, null)).MustNotHaveHappened();
```

Comment:
the ExportSimulationRunner deals with one project at a time so we don't have to do that there